proto/osmosis/tokenfactory/v1beta1/authorityMetadata.proto

```diff
@@ -4,7 +4,7 @@ package osmosis.tokenfactory.v1beta1;
 import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
 
-option go_package = "github.com/osmosis-labs/osmosis/v12/x/tokenfactory/types";
+option go_package = "github.com/CosmWasm/token-factory/x/tokenfactory/types";
```

Comment:
👍 makes sense. 
Can you also move the files into the right folder, too? See my `module` branch for other proto changes.